database/testhelpers/test_helpers.go

```diff
@@ -0,0 +1,100 @@
+package testhelpers
+
+import (
+	"os"
+	"reflect"
+
+	"github.com/thrasher-corp/gocryptotrader/database"
+	"github.com/thrasher-corp/gocryptotrader/database/drivers"
+	psqlConn "github.com/thrasher-corp/gocryptotrader/database/drivers/postgres"
+	sqliteConn "github.com/thrasher-corp/gocryptotrader/database/drivers/sqlite3"
+)
+
+var (
+	// TempDir temp folder for sqlite database
+	TempDir string
+	// PostgresTestDatabase postgresql database config details
+	PostgresTestDatabase *database.Config
+)
+
+// GetConnectionDetails returns connection details for CI or test db instances
+func GetConnectionDetails() *database.Config {
+	_, exists := os.LookupEnv("TRAVIS")
+	if exists {
+		return &database.Config{
+			Enabled: true,
+			Driver:  "postgres",
+			ConnectionDetails: drivers.ConnectionDetails{
+				Host:     "localhost",
+				Port:     5432,
+				Username: "postgres",
+				Password: "",
+				Database: "gct_dev_ci",
+				SSLMode:  "",
+			},
+		}
+	}
+
+	_, exists = os.LookupEnv("APPVEYOR")
+	if exists {
+		return &database.Config{
+			Enabled: true,
+			Driver:  "postgres",
+			ConnectionDetails: drivers.ConnectionDetails{
+				Host:     "localhost",
+				Port:     5432,
+				Username: "postgres",
+				Password: "Password12!",
+				Database: "gct_dev_ci",
+				SSLMode:  "",
+			},
+		}
+	}
+
+	return &database.Config{
+		Enabled: true,
+		Driver:  "postgres",
+		ConnectionDetails: drivers.ConnectionDetails{
+			Host:     "localhost",
+			Port:     5432,
+			Username: "andrew",
```

Comment:
Although I'd' vote Andrew for president, how about gctuser

Reply:
will remove these not meant to be there as user should self configure